requirements-dev.txt

```diff
@@ -1,48 +1,65 @@
 # DO NOT EDIT THIS FILE DIRECTLY - use ./script/update-requirements to update
-Pygments==2.18.0
+PyYAML==6.0.2
+Pygments==2.19.1
+SecretStorage==3.3.3
 black==24.10.0
 build==1.2.2.post1
-certifi==2024.8.30
+certifi==2025.6.15
 cffi==1.17.1
-charset-normalizer==3.3.2
-click==8.1.7
-cmarkgfm==2024.1.14
-coverage==7.6.1
+charset-normalizer==3.4.2
+click==8.2.1
+cmarkgfm==2024.11.20
+coverage==7.9.1
+cryptography==45.0.4
+dnspython==2.7.0
 docutils==0.21.2
-importlib_metadata==8.5.0
-iniconfig==2.0.0
-isort==5.13.2
+fqdn==1.5.1
+id==1.5.0
+idna==3.10
+iniconfig==2.1.0
+isort==6.0.1
 jaraco.classes==3.4.0
 jaraco.context==6.0.1
 jaraco.functools==4.1.0
-keyring==25.4.1
+jeepney==0.9.0
+keyring==25.6.0
 markdown-it-py==3.0.0
+mdformat==0.7.22
+mdformat_footnote==0.1.1
+mdformat_frontmatter==2.0.8
+mdformat_myst==0.2.1
+mdformat_tables==1.0.0
+mdit-py-plugins==0.4.2
 mdurl==0.1.2
-more-itertools==10.5.0
-mypy-extensions==1.0.0
-nh3==0.2.18
-packaging==24.1
+more-itertools==10.7.0
+mypy_extensions==1.1.0
+natsort==8.4.0
+nh3==0.2.21
+packaging==25.0
 pathspec==0.12.1
-pkginfo==1.10.0
-platformdirs==4.3.6
-pluggy==1.5.0
+platformdirs==4.3.8
+pluggy==1.6.0
 pprintpp==0.4.0
 pycountry-convert==0.7.2
 pycountry==24.6.1
 pycparser==2.22
-pyflakes==3.2.0
+pyflakes==3.3.2
 pyproject_hooks==1.2.0
-pytest-cov==5.0.0
-pytest-mock==3.14.0
-pytest==8.3.3
+pytest-cov==6.2.1
+pytest-mock==3.14.1
+pytest==8.4.0
 pytest_network==0.0.1
+python-dateutil==2.9.0.post0
 readme_renderer==44.0
 repoze.lru==0.7
 requests-toolbelt==1.0.0
-requests==2.32.3
+requests==2.32.4
 rfc3986==2.0.0
-rich==13.9.2
-twine==5.1.1
-urllib3==2.2.3
-wheel==0.44.0
-zipp==3.20.2
+rich==14.0.0
+ruamel.yaml.clib==0.2.12
+ruamel.yaml==0.18.14
```

Comment:
What OS and python are you working on? Seeing a bit more change here than I'd expect from an update and wondering if it's different requirements on different envs.